openfisca_core/columns.py

```diff
@@ -218,6 +218,13 @@ class DateCol(Column):
     json_type = 'Date'
     val_type = 'date'
 
+    def __init__(self, default = None, **kwargs):
+        super(DateCol, self).__init__(**kwargs)
+        if default is None:
+            default = datetime.date.fromtimestamp(0)  # 0 == 1970-01-01
```

Comment:
What about returning by default `date.max` (or `date.min`) ? 1970 sounds a lot like a real date, which can be confusing.
Just a suggestion, otherwise the PR seems good to me.